ush/config.sh.RRFS_wcoss

```diff
@@ -0,0 +1,150 @@
+    BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}0000 ${DATE_LAST_CYCL}2300 06:00:00"
+  else
+    PREEXISTING_DIR_METHOD="overwrite"
+    PREP_COLDSTART_CYCLEDEF="00 03,15 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+    PREP_WARMSTART_CYCLEDEF="00 00-02,04-14,16-23 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+    ANALYSIS_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+    FORECAST_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+    POSTPROC_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+    POSTPROC_LONG_CYCLEDEF="00 00,06,12,18 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  fi
+else
+  INITIAL_CYCLEDEF="00 03,15 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  BOUNDARY_CYCLEDEF="00 00,06,12,18 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  PREP_COLDSTART_CYCLEDEF="00 03,15 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  PREP_WARMSTART_CYCLEDEF="00 00-02,04-14,16-23 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  ANALYSIS_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  FORECAST_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  POSTPROC_CYCLEDEF="00 00-23/01 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  POSTPROC_LONG_CYCLEDEF="00 00,06,12,18 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+  ARCHIVE_CYCLEDEF="00 07 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
+fi
+
+FCST_LEN_HRS="12"
+POSTPROC_LEN_HRS="3"
+POSTPROC_LONG_LEN_HRS="12"
+FCST_LEN_HRS_CYCLES=(12 3 3 06 3 3 12 3 3 6 3 3 12 3 3 6 3 3 12 3 3 6 3 3)
+for i in {0..23}; do FCST_LEN_HRS_CYCLES[$i]=3; done
+for i in {0..23..6}; do FCST_LEN_HRS_CYCLES[$i]=12; done
+DA_CYCLE_INTERV="1"
+RESTART_INTERVAL="1 2 3 4 5 6"
+
+NCORES_RUN_ANAL=240
+QUEUE_ANALYSIS="dev"
+QUEUE_HPSS="dev"
+HALO_BLEND=10
+
+if [[ -n $RESERVATION ]] ; then
+  NNODES_MAKE_LBCS="3"
+  PPN_MAKE_ICS="20"
+  NNODES_MAKE_LBCS="3"
+  PPN_MAKE_LBCS="20"
+  NNODES_RUN_POST="1"
+  PPN_RUN_POST="40"
+fi
+
+WTIME_RUN_FCST="02:45:00"
+
+EXTRN_MDL_NAME_ICS="FV3GFS"
+EXTRN_MDL_NAME_LBCS="FV3GFS"
+FV3GFS_FILE_FMT_ICS="grib2"
+FV3GFS_FILE_FMT_LBCS="grib2"
+
+envir="para"
+
+NET="RRFS_CONUS"
+TAG="RRFS_dev2_CONUS"
+
+OBSPATH="/gpfs/hps/nco/ops/com/rap/prod"
+ENKF_FCST="/gpfs/dell1/nco/ops/com/gfs/prod"
+#USE_CUSTOM_POST_CONFIG_FILE="TRUE"
+#CUSTOM_POST_CONFIG_FP="/mnt/lfs4/BMC/nrtrr/RRFS/dev1-ufs-srweather-app/src/EMC_post/parm/postxconfig-NT-fv3lam_rrfs.txt"
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_dev1"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/gpfs/hps3/ptmp/Ming.Hu/ptmp"
+NCL_REGION="conus"
+MODEL="RRFS_dev2"
+
+#
+# In NCO mode, the following don't need to be explicitly set to "FALSE" 
+# in this configuration file because the experiment generation script
+# will do this (along with printing out an informational message).
+#
+#RUN_TASK_MAKE_GRID="FALSE"
+#RUN_TASK_MAKE_OROG="FALSE"
+#RUN_TASK_MAKE_SFC_CLIMO="FALSE"
+
+RUN="RRFS_dev2"
+COMINgfs=""
+
+STMP="/gpfs/hps3/ptmp/Ming.Hu/stmp"  # Path to directory STMP that mostly contains input files.
+PTMP="/gpfs/hps3/ptmp/Ming.Hu/ptmp"  # Path to directory STMP that mostly contains input files.
```

Comment:
Please leave out personal output locations for retro runs.

Reply:
There are no official space on WCOSS. All tests (even parallel for operation updates) are done in person space.